src/diagnostics/default_diagnostics.jl

```diff
@@ -186,6 +186,24 @@ Return a `ScheduledDiagnostics` that computes the hourly average for the given v
 hourly_average(short_names; output_writer = HDF5Writer()) =
     hourly_averages(short_names; output_writer)[1]
 
+"""
+    tenmin_insts(short_names...; output_writer = HDF5Writer())
+
+
+Return a list of `ScheduledDiagnostics` that compute the 10 min instantaneous output for the given variables.
+"""
+tenmin_insts(short_names...; output_writer = HDF5Writer()) =
+    common_diagnostics(10 * 60, nothing, output_writer, short_names...)
```

Comment:
I think that this will return in some warnings being raised. With `common_diagnostics` you set `output_every`, but `compute_every` is set to `:timestep`. If you don't want time reductions, `compute_every` will be adjusted to be the same as `output_every`, but a warning will be produced.

So, I think that you have to change this to
```suggestion
function tenmin_insts(short_names...; output_writer = HDF5Writer()) 
period = 10 * 60 # seconds
        return [
        ScheduledDiagnosticTime(
            variable = get_diagnostic_variable(short_name),
            compute_every = period,
            output_every = period, 
            output_writer = output_writer,
        ) for short_name in short_names
    ]

end
```

Reply:
Thanks!